inst/report/Report_KRI.Rmd

```diff
@@ -126,7 +126,7 @@ Report_OverviewText(
 
 ```{r, echo=FALSE, results='asis'}
 
-for (i in unique(params$dfMetrics$MetricID)) {
+for (i in unique(params$dfResults$MetricID)) {
```

Comment:
I think this really is the simplest way to fix it @lauramaxwell @samussiah , other "checks" may include making sure the metrics dataset coming out of `3_reporting/metrics.yaml` only has metric IDs of the results?

Reply:
oh nice- i think this makes sense! way easier than what i was thinking we'd have to do 🎉

Reply:
Unless we want to put in additional guardrails, I think this is actually it

Reply:
Makes sense to me!